### PR TITLE
add future chromeos compatablility

### PR DESCRIPTION
**Pull request by koolkid327**
Fixes # copy glibc to /usr/local

## Additional information

Works properly:
- [x] `x86_64`
- [x] `i686`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_REPO=https://github.com/koolkid327/chromebrew.git CREW_BRANCH=patch-1 crew update
```

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -151,7 +151,7 @@ function curl () {
 }
 
 # This will create the directories.
-crew_folders="bin cache doc docbook include lib/crew/packages lib$LIB_SUFFIX libexec man sbin share var etc/crew/meta etc/env.d tmp/crew/dest"
+crew_folders="lib64 bin cache doc docbook include lib/crew/packages lib$LIB_SUFFIX libexec man sbin share var etc/crew/meta etc/env.d tmp/crew/dest"
```
Comment (uberhacker) [inline install.sh:154]: `$LIB_SUFFIX` abstracts the library folder which is not `lib64` on arm and i686 architecture.  Is `$LIB_SUFFIX` not set to `64` on your system?

File: install.sh [hunk 2]
```diff
@@ -169,6 +169,9 @@ BOOTSTRAP_PACKAGES='zstd crew_mvdir ruby git ca_certificates libyaml openssl'
 [[ "${ARCH}" == "i686" ]] && BOOTSTRAP_PACKAGES+=' zlibpkg gcc_lib'
 
 if [[ -n "${CHROMEOS_RELEASE_CHROME_MILESTONE}" ]]; then
+  sudo cp /lib64/libc.so* /usr/local/
+  sudo chown chronos /usr/local/lib64/libc.so*
+  sudo chmod 777 /usr/local/lib64/libc.so*
```
Comment (uberhacker) [inline install.sh:174]: This only works for x86_64 architecture and we need to abstract `/usr/local` with `$CREW_PREFIX`.  What issue are you experiencing to require this change?  Maybe an issue should be opened to further explain your installation trouble?

File: install.sh (hunk 1 above)
Comment (koolkid327) [reply install.sh]: done!

File: install.sh (hunk 2 above)
Comment (koolkid327) [reply install.sh]: the one that you just fixed

File: install.sh [hunk 3]
```diff
@@ -169,9 +169,9 @@ BOOTSTRAP_PACKAGES='zstd crew_mvdir ruby git ca_certificates libyaml openssl'
 [[ "${ARCH}" == "i686" ]] && BOOTSTRAP_PACKAGES+=' zlibpkg gcc_lib'
 
 if [[ -n "${CHROMEOS_RELEASE_CHROME_MILESTONE}" ]]; then
-  sudo cp /lib$LIB_SUFFIX/libc.so* /usr/local/
-  sudo chown chronos /usr/local/lib$LIB_SUFFIX/libc.so*
-  sudo chmod 777 /usr/local/lib$LIB_SUFFIX/libc.so*
+  sudo cp /lib$LIB_SUFFIX/libc.so* $CREW_PREFIX/lib$LIB_SUFFIX
+  sudo chown chronos $CREW_PREFIX/lib$LIB_SUFFIX/libc.so*
+  sudo chmod 777 $CREW_PREFIX/lib$LIB_SUFFIX/libc.so*
```
Comment (uberhacker) [inline install.sh:174]: Can we change the 777 to 644?  These libraries should not need execute or other group write permission.

Comment (koolkid327) [reply install.sh:174]: oh

Comment (uberhacker) [reply install.sh:174]: What are your thoughts about this proposed change @satmandu and @supechicken?

Comment (koolkid327) [reply install.sh]: so that we will have to make less every time google makes 100 chromeos updates

File: install.sh (hunk 1 above)
Comment (satmandu) [reply install.sh]: With `lib$LIB_SUFFIX` we don't need `lib64`...

**Review by satmandu — APPROVED** [review]
Seems like it shouldn't break anything...

**Comment by uberhacker** [comment]
@koolkid327:  Not sure if you noticed but shellcheck is complaining about missing double quotes.

**Review by uberhacker — APPROVED** [review]

**Pull request merged**